### PR TITLE
Add total and unread counts to modding watchlist

### LINKED ISSUE #4148: add info to the modding watchlist page header
image
currently, the modding watchlist header has a no-height box on the right side
so my proposal is:
add information to it, just like 
image
the forum subscriptions header states the number of subscribed threads, and the number of ones with new replies.
on the modding discussions page, it could be better if the box was used to stay the number of subscribed beatmaps and the number of ones with new replies!

### PR DESCRIPTION
**Pull request by cl8n**
resolves #4148

### CODE REVIEW

File: app/Models/BeatmapsetWatch.php [hunk 1]
```diff
@@ -65,6 +65,11 @@ public function scopeRead($query)
         });
     }
 
+    public function scopeUnread($query)
+    {
+        $query->whereColumn('last_read', '<', 'last_notified');
```
Comment (nanaya) [inline app/Models/BeatmapsetWatch.php:70]: Nothing currently updates `last_notified` so the result is wrong.

**Review by nanaya — CHANGES_REQUESTED** [review]

File: app/Models/BeatmapsetWatch.php (hunk 1 above)
Comment (cl8n) [reply app/Models/BeatmapsetWatch.php:70]: huh... I just flipped scopeRead/isRead expecting it to work, how does `isRead()` have the right result on live site?

Comment (nanaya) [reply app/Models/BeatmapsetWatch.php:70]: It's not currently used anywhere.

Comment (nanaya) [reply app/Models/BeatmapsetWatch.php:70]: er, sorry, it is used somewhere. But it's currently wrong as the original thing that updates the column has been replaced with something else.

**Review by nanaya — APPROVED** [review]

**Pull request merged**